hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStore.java

```diff
@@ -94,6 +97,14 @@ public RDBStore(File dbFile, DBOptions options,
     this.writeOptions = writeOptions;
 
     try {
+      List<TableConfig> columnFamiliesInDb = getColumnFamiliesInExistingDb();
+      List<TableConfig> extraCf = columnFamiliesInDb.stream().filter(
+          cf -> !families.contains(cf)).collect(Collectors.toList());
+      for (TableConfig family : extraCf) {
+        LOG.info("Adding column family {}", family.getName());
```

Comment:
One question, can these extra column families can be dropped, if they are not required after upgrade/downgrade.

Because now we removed S3Table in #940. So, once after upgrade, technically we don't require the column family any more.

Reply:
@bharatviswa504 That is a good point. Based on our offline discussion, this will be handled on a case-by-case basis as outlined in the Ozone upgrades design doc (HDDS-3698).